resubmit.py

```diff
@@ -47,41 +45,31 @@ def modifySchema(helper, workflow, user, group, cache, events, firstLumi, backfi
     if backfill is True, modifies RequestString, ProcessingString, AcquisitionEra
     and Campaign to say Backfill, and restarts requestDate.
     """
+    # list of keys to be removed from the new request schema
+    paramBlacklist = ['AcquisitionEra', 'BlockCloseMaxEvents', 'BlockCloseMaxFiles', 'BlockCloseMaxSize',
+                      'BlockCloseMaxWaitTime', 'CouchURL', 'CouchWorkloadDBName', 'CustodialGroup',
+                      'CustodialSubType', 'Dashboard', 'GracePeriod', 'Group', 'HardTimeout', 'InitialPriority',
+                      'inputMode', 'MaxMergeEvents', 'MaxMergeSize', 'MaxRSS', 'MaxVSize', 'MinMergeSize',
+                      'NonCustodialGroup', 'NonCustodialSubType', 'ProcessingString', 'OutputDatasets',
+                      'ReqMgr2Only', 'RequestDate' 'RequestorDN', 'RequestName', 'RequestStatus',
+                      'RequestTransition', 'RequestWorkflow', 'SiteWhitelist', 'SoftTimeout',
+                      'SoftwareVersions', 'SubscriptionPriority', 'Team', 'timeStamp']
+
     result = {}
-    for (key, value) in helper.data.request.schema.dictionary_whole_tree_().items():
-        #previous versions of tags
-        if key == 'ProcConfigCacheID':
-            result['ConfigCacheID'] = value
-        elif key == 'RequestSizeEvents':
-            result['RequestSizeEvents'] = value
-        #Set memory
-        elif key == 'Memory':
-            if memory != None:
-                result['Memory'] = memory
-            else:
-                result['Memory'] = value
-        #requestor info
+    for key, value in helper.data.request.schema.dictionary_whole_tree_().items():
+        if not value or key in paramBlacklist:
```

Comment:
including this would be a problem if value is False... I'll update the code soon